tensorboard/components/vz_example_viewer/vz-example-viewer.ts

```diff
@@ -0,0 +1,1497 @@
+// TODO(jwexler): Do we need these deps?
+//goog.require('goog.crypt.base64');
+//goog.require('jspb.Map');
+
+import BytesList from 'goog:proto.tensorflow.BytesList'; // from //third_party/tensorflow/core:protos_all_jspb_proto
+import Example from 'goog:proto.tensorflow.Example'; // from //third_party/tensorflow/core:protos_all_jspb_proto
```

Comment:
The comments shouldn't be necessary since folks can grep.

Reply:
done, were initially auto-generated by taze